inference-engine/src/mkldnn_plugin/utils/node_dumper.cpp

```diff
@@ -130,56 +133,77 @@ void NodeDumper::dumpInternalBlobs(const MKLDNNNodePtr& node) const {
 
 void NodeDumper::dump(const BlobDumper& bd, const std::string& file) const {
     switch (dumpFormat) {
-    case DUMP_FORMAT::BIN: {
+    case FORMAT::BIN: {
         bd.dump(file);
         break;
     }
-    case DUMP_FORMAT::TEXT: {
+    case FORMAT::TEXT: {
         bd.dumpAsTxt(file);
         break;
     }
     default:
-        IE_THROW() << "Unknown dump format";
+        IE_THROW() << "NodeDumper: Unknown dump format";
     }
 }
 
-bool NodeDumper::shouldBeDumped(const MKLDNNNodePtr& node) const {
+bool NodeDumper::shouldBeDumped(const MKLDNNNodePtr& node, const std::string& portsKind) const {
     if (dumpFilters.empty())
         return false;
 
-    if (dumpFilters.count(FILTER::BY_EXEC_ID)) { // filter by exec id env set
+    if (dumpFilters.count(FILTER::BY_PORTS)) { // filter by ports configured
+        if (dumpFilters.at(FILTER::BY_PORTS) != "BOTH" &&
```

Comment:
Maybe it makes sense to use enum instead of hardcoded string constants?

Reply:
The same answer here.